apps/condo/domains/common/components/PhoneInput.tsx

```diff
@@ -64,7 +67,7 @@ export const PhoneInput: React.FC<IPhoneInputProps> = forwardRef((props, ref) =>
     const { value, placeholder, style, disabled, block, showCountryPrefix = true, ...otherProps } = props
     const configSize = useContext<SizeType>(ConfigProvider.SizeContext)
     const { organization } = useOrganization()
-    const userOrganizationCountry = showCountryPrefix && get(organization, 'country', 'ru')
+    const userOrganizationCountry = showCountryPrefix && get(organization, 'country', defaultLocale)
```

Comment:
What if there's not config var?
```suggestion
    const userOrganizationCountry = showCountryPrefix && get(organization, 'country', defaultLocale || 'en')
```

Reply:
We already have a fallback for that.